### DISCUSSION TITLE
Fluent UI React charts

### DISCUSSION THREAD

**Discussion by Sagensagen** [sydsutton/FS.FluentUI#7] [Ideas]
Hi, love these bindings and have been using them quite a lot lately.

Is there given any though about the Fluent UI React charts as a part of this package? I would love to be able to use the [charts](https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/GroupedVerticalBarChart#Overview) as a part of new applications along these components to get the consistent design throughout the apps. 

If not i could probably give it a shot if i got time to spare.

Cheers, and good work so far! :rocket:

**Comment by sydsutton** [comment]
Thanks, and glad you're enjoying using FS.FluentUI! I hadn't considered creating bindings for FluentUI React charts-- Although I think it would probably be way easier than the component library, looking at the npm downloads, it doesn't seem like it would be worth the work. It only has an average of 3 weekly downloads, and people can just as easily use Feliz.Recharts. If JS/TS people aren't really using it, F# people probably won't use it either. That's just my take on it at least. 

Thanks again!

**Reply by Sagensagen** [reply]
I see, and support your thought here regarding the cost/usage of writing bindings. Our goto charting lib has been feliz.Recharts, but i do not like the feel of it even though it does the job. From what i saw on npm it is about 3k downloads/week, which is somewhat better :)

Feel free to close this discussion as it's intention was to highlight your take on it.

**Reply by sydsutton** [reply]
Sounds good, thanks for the suggestion @Sagensagen!

**Reply by sydsutton** [reply]
@Sagensagen once FluentUI charts in their TS docs move from Preview to Stable, I do plan on making a separate project for those bindings as well. Hopefully they don't add a lot more charts-- not sure if I can handle the workload. 
https://react.fluentui.dev/?path=/docs/charts_introduction--docs
